### PR TITLE
feat: postcss autoprefixer

### PR DESCRIPTION
**Pull request by juliodialpad**
## Description

- Added [PostCSS autoprefixer](https://github.com/postcss/autoprefixer) to enhance compatibility with all the supported browsers
- Cleaned up dialtone-docs.less, dialtone-syntax.less
- Updated fixed values to CSS variables
- Disabled VuePress autoprefixer over dialtone.css file. dialtone.css file still has it's own autoprefix configuration running.
- Added Dialtone's VuePress theme files to stylelint.

## Pull Request Checklist

 - [x] Ask the contributors if a similar effort is already in process or has been solved.
 - [x] Review the [contribution guidelines](https://github.com/dialpad/dialtone/blob/staging/.github/CONTRIBUTING.md).
 - [ ] Use `staging` as your pull request's base branch. (All PRs using `production` as its base branch will be declined).
 - [x] Ensure all `gulp` scripts successfully compile.
 - [ ] Update, remove, or extend all affected documentation.
 - [ ] Ensure no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).

### CODE REVIEW

File: docs/.vuepress/theme/assets/css/dialtone.css [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/* autoprefixer: off */
+@import "../../../../../lib/dist/css/dialtone.css";
```
Comment (iropolo) [inline docs/.vuepress/theme/assets/css/dialtone.css]: could we use `@` alias? would work?

**Review by iropolo — APPROVED** [review]

File: docs/.vuepress/theme/assets/css/dialtone.css (hunk 1 above)
Comment (juliodialpad) [reply docs/.vuepress/theme/assets/css/dialtone.css]: No, unfortunately it doesn't work on CSS files

Comment (braddialpad) [reply docs/.vuepress/theme/assets/css/dialtone.css]: I think historically aliases never worked in css files... however looks like they do with vite https://vitejs.dev/guide/features.html#import-inlining-and-rebasing

So I think we can make this change

**Review by braddialpad — APPROVED** [review]
Pre-approving for after alias path is fixed.

File: docs/.vuepress/theme/assets/css/dialtone.css (hunk 1 above)
Comment (juliodialpad) [reply docs/.vuepress/theme/assets/css/dialtone.css]: Nice! I didn't new I could use the vite aliases, thanks @braddialpad! updated

**Pull request merged**